grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java

```diff
@@ -72,6 +72,9 @@
     private static final String MATCH_KEYWORD = "match";
 
     public ImportController(ClusterManager manager) {
+        if (manager==null) {
```

Comment:
This manager check might be better places in a common area to be shared between `TasksController` and `ImportController`. A utility method maybe?